nautobot/ipam/tests/test_api.py

```diff
@@ -197,7 +197,13 @@ def setUpTestData(cls):
 
```

Comment:
To fix the test, we should probably create *new* VRFs to assign to, instead of relying on the assumption that the existing VRFs don't already have an assignment to specific existing Devices/VDCs/VMs.